### PR TITLE
RO update Nov, 08 2015

### PR DESCRIPTION
**Pull request by bogdangorescu**
Added measurement units, name of Care Portal, drag and drop food, In the future.

### CODE REVIEW

File: lib/language.js [hunk 1]
```diff
@@ -4389,29 +4389,37 @@ function init() {
       ,it: 'Lingua'
       }
     ,'Add new' : {
-      cs: 'Přidat nový'
+      cs: 'Přidat nový',
+      ,ro: 'Adaugă'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1285468559/issues/source?bid=2452379&fileBranchId=2945886#l4393)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1285468559/issues/source?bid=2452379&fileBranchId=2945886#l4393)

**Comment by jasoncalabrese** [comment]
Look like you have a double ,

File: lib/language.js [hunk 2]
```diff
@@ -4389,29 +4389,37 @@ function init() {
       ,it: 'Lingua'
       }
     ,'Add new' : {
-      cs: 'Přidat nový'
+      cs: 'Přidat nový',
+      ,ro: 'Adaugă nou'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1287885599/issues/source?bid=2452379&fileBranchId=2945886#l4393)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1287885599/issues/source?bid=2452379&fileBranchId=2945886#l4393)

**Comment by bogdangorescu** [comment]
Finally!...
:)


B

On Mon, Nov 9, 2015 at 8:24 PM, Codecov <notifications@github.com> wrote:

> Current coverage
> <https://codecov.io/github/nightscout/cgm-remote-monitor?ref=6c6df44163acc6451a5348ba0753adbd842716bb>
> is 77.78%
>
> Merging *#1338
> <https://github.com/nightscout/cgm-remote-monitor/pull/1338>* into *dev*
> will increase coverage by *+0.03%* as of 6c6df44
> <https://codecov.io/github/nightscout/cgm-remote-monitor/commit/6c6df44163acc6451a5348ba0753adbd842716bb>
>
> @@              dev   #1338   diff @@
> =====================================
>   Files          85      85
>   Stmts        7053    7063    +10
>   Branches        0       0
>   Methods         0       0
> =====================================+ Hit          5484    5494    +10
>   Partial         0       0
>   Missed       1569    1569
>
> Review entire Coverage Diff
> <https://codecov.io/github/nightscout/cgm-remote-monitor/compare/ca61bddffbeb954758900d252e53dafa97b58fb4...6c6df44163acc6451a5348ba0753adbd842716bb>
> as of 6c6df44
> <https://codecov.io/github/nightscout/cgm-remote-monitor/commit/6c6df44163acc6451a5348ba0753adbd842716bb>
>
> ------------------------------
> Uncovered Suggestions
> <https://codecov.io/github/nightscout/cgm-remote-monitor/features/suggestions?ref=6c6df44163acc6451a5348ba0753adbd842716bb>
>
>    1. +0.34% via .../client/boluscalc.js#151...174
>    <https://codecov.io/github/nightscout/cgm-remote-monitor/lib/client/boluscalc.js?ref=6c6df44163acc6451a5348ba0753adbd842716bb#151>
>    2. +0.28% via .../client/boluscalc.js#294...313
>    <https://codecov.io/github/nightscout/cgm-remote-monitor/lib/client/boluscalc.js?ref=6c6df44163acc6451a5348ba0753adbd842716bb#294>
>    3. +0.26% via .../client/boluscalc.js#253...270
>    <https://codecov.io/github/nightscout/cgm-remote-monitor/lib/client/boluscalc.js?ref=6c6df44163acc6451a5348ba0753adbd842716bb#253>
>    4. *See 7 more...
>    <https://codecov.io/github/nightscout/cgm-remote-monitor/features/suggestions?ref=6c6df44163acc6451a5348ba0753adbd842716bb>*
>
> Powered by Codecov <https://codecov.io>. Updated on successful CI builds.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nightscout/cgm-remote-monitor/pull/1338#issuecomment-155146643>
> .
>



-- 
Bogdan Gorescu

**Pull request merged**